manifests/charts/base/files/gen-istio-cluster.yaml

```diff
@@ -5767,8 +5767,6 @@ rules:
   - apiGroups: ["certificates.k8s.io"]
     resources:
       - "signers"
-    resourceNames:
```

Comment:
giving ability to approve any CSR seems a bit dangerous...

Reply:
I think i understand the intention of this fix, when user specifies another signer, it does not work without this.

Reply:
Given istiod will not monitor all k8s CSR - it just get the csr from istio-agent and generate a k8s CSR accordingly. So  it should be ok for us to grant istiod authority to approve any signers.

Reply:
@howardjohn - could you elaborate on why you think this is dangerous? 

@irisdingbj - does any RFC highlight how the additional signers will be configured for istiod?

Reply:
This is basically cluster root. This means someone can create ANY CSR, with  any contents, and get it approved by Istiod, right?

Are there any other projects giving such a broad permission?

Reply:
I assumed that since CSR requests are proxied through Istiod, it will be istiod's responsibility to ensure that
(a) the certificates requested don't request more usages/subjects/alt-names that expected
(b) Istiod is able to successfully authenticate all the entities that request certificates.

However, on second thought, this does confer too much responsibility to Istiod. Better to limit Istiod's approval power to a. more limited set of signers that can be configured at install time.

Reply:
I agree that Istiod *should* limit the requests, but its good to reduce privs where possible like you mentioned, for a lower attack surface.

Reply:
@shankgan https://docs.google.com/document/d/1rxTSLuepcE_OJHj4C23RMJcgyIZzlBI-GqO8Y37SPko/edit#heading=h.qlmnvhzb2ib2 proposing add `cert_signer_domain ` into meshconfig